### PR TITLE
Added dev.okteto.com/namespace label to external resource objects when they are being created

### PR DESCRIPTION
**Pull request by ifbyol**
Partially Fixes # https://github.com/okteto/app/issues/6269

Include the label `dev.okteto.com/namespace` in the external resources when they are created/updated as that would help us to list all external resources within different namespaces in mor efficient way

### CODE REVIEW

File: pkg/externalresource/control.go [hunk 1]
```diff
@@ -99,9 +99,13 @@ func translate(name string, externalResource *ExternalResource) *k8s.External {
 			APIVersion: fmt.Sprintf("%s/%s", k8s.GroupName, k8s.GroupVersion),
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: format.ResourceK8sMetaString(name),
+			Name:      format.ResourceK8sMetaString(name),
+			Namespace: ns,
```
Comment (ifbyol) [inline pkg/externalresource/control.go:103]: We were not specifying the namespace but now that we have it, it doesn't hurt to include it

**Review by AdrianPedriza — APPROVED** [review]
LGTM!

File: pkg/constants/constants.go [hunk 2]
```diff
@@ -85,4 +85,7 @@ const (
 
 	// OktetoGitCommitEnvVar is the SHA1 hash of the last commit of the branch.
 	OktetoGitCommitEnvVar = "OKTETO_GIT_COMMIT"
+
+	// OktetoNamespaceLabel is the label used to identify the namespace where the resource lives
+	OktetoNamespaceLabel = "dev.okteto.com/namespace"
```
Comment (jLopezbarb) [inline pkg/constants/constants.go:90]: What's the idea with this new constant? Should we include it in every resource? Or just on CRDs?

Comment (ifbyol) [reply pkg/constants/constants.go:90]: Just for the external resources or any object in which we want to do queries by that. In this case, it is to optimize how we retrieve the endpoints for external resources from a list of namespaces

**Review by jLopezbarb — APPROVED** [review]

**Pull request merged**